src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,221 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> first;
+    private Node<T> last;
```

Comment:
Don't forget about access modifiers. Fields `size`, `first`, and `last` should be private to prevent direct access.